bin/test_access

```diff
@@ -79,7 +79,16 @@ echo %%% vlan-assigned | tee -a $TEST_RESULTS
 send_udp forch-faux-2 "192.168.1.254/68" 5
 send_udp forch-faux-3 "192.168.1.254/67" 5
 tail inst/forch-faucet-1/forch.log
+
+RESPONSES=$( printf "%.0f" $(wget 0.0.0.0:8302 -O- | grep radius_query_responses_total | grep -v 'HELP\|TYPE' | awk '{print $2'}) )
```

Comment:
I would move this and the other one in tho the shared functions file, and then check both (responses and timeouts) in both places (with a simple function call).  Same exact code, more or less, just factored out into a function and used in 2x places.